data_handler.py

```diff
@@ -295,5 +295,17 @@ def show_one_user(user_id):
     return user_data
 
 
+def get_tags_with_questions():
+    sql_data = DAL.tags.get_tags_with_questions()
+    data = {}
+    for item in sql_data:
+        if item['name'] not in data:
+            data[item['name']] = [item]
+        else:
+            data[item['name']] += [item]
+    print(data)
```

Comment:
Can You please remove this `print` statement?

Reply:
no

Reply:
# :cry: